test/integration/lifecycle.js

```diff
@@ -228,7 +228,7 @@ function runLifecycle(testMatrix) {
             'parameters':service.updatingParameters
           })
           .end(function (err, res) {
-            res.should.have.status(200);
+            res.status.should.be.oneOf([200, 202]);
```

Comment:
Seems 201 is also valid according to the [spec](https://github.com/openservicebrokerapi/servicebroker/blob/v2.14/spec.md#response-6)?